data_manager_select_index_by_path/data_manager/test_path_name_value_key_manager.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+import pytest
+import json
+from pathlib import Path
+
+from path_name_value_key_manager import DataTable, check_tab
+
+indexes_yml = Path(__file__).parent / "indexes.yml"
+test_data = Path(__file__).parent / "../test-data"
+
+
+def test_check_tab():
+    check_tab("test", "This text does not contain a tab and succeeds")
+
+
+@pytest.mark.xfail
```

Comment:
This _skips_ a failing test, it does not check whether the failure actually occurs. 

To check for exceptions, use:

```
with pytest.raises(ExpextedExceptionClass):
     my_func_that_errors_out(...)
```